src/plugin/class-engine.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace DotOrg\TryWordPress;
+
+class Engine {
+
+	public function __construct() {
+		require 'class-post-type-ui.php';
+		require 'class-promoter.php';
+		require 'class-meta-fields-manager.php';
+		require 'class-rest-api-extender.php';
+		require 'class-storage.php';
+
+		( function () {
+			$custom_post_type = Storage::POST_TYPE;
```

Comment:
Just out of curiosity, what's the purpose of wrapping this code in an anonymous function?

Reply:
No reason, it was just copy-pasted from `plugin.php` where the purpose was to avoid the declared variables polluting the global namespace. The next simplification PR would take care of it, when a bunch of then unused stuff would be removed.